examples/todoapp/gradle.properties

```diff
@@ -23,6 +23,5 @@ org.gradle.parallel=true
 org.gradle.caching=true
 kotlin.native.disableCompilerDaemon=true
 
-#todo remove -Pkotlin.version=1.6.20 from Xcode project, when stable version on Compose with Koltin 1.6.20 or later released
-kotlin.version=1.6.10
-compose.version=1.1.0
\ No newline at end of file
+kotlin.version=1.7.0
+compose.version=1.2.0-alpha01-dev745
```

Comment:
Thanks! But let's keep this PR until we release a stable version of 1.2.0. The general rule for `/examples` and `/tutorials` is to use a stable version of Compose.

Reply:
1.2.0 is out. We can merge `master` to this PR, solving conflicts (new versions are already there), and the merge this PR to `master`

Reply:
Sure, I will update the PR soon.

Reply:
I have pushed updates, also updated the PR description.